### PR TITLE
Charge stat for Item

### PR DESCRIPTION
**Pull request by sk7725**
+ Makes electric discharge, similar to batteries, when destroyed.

- [x] Add it to surge alloy
- [x] Tweak onDestroyed()
- [x] Add Stat.discharge
- [x] Bundle mundle
- [x] Rename to "charge"
- [x] Surge unit explosions

### CODE REVIEW

File: core/src/mindustry/type/Item.java [hunk 1]
```diff
@@ -17,6 +17,8 @@
     public float flammability = 0f;
     /** how radioactive this item is. 0=none, 1=chernobyl ground zero */
     public float radioactivity;
+    /** how electrically potent this item is.*/
```
Comment (summetdev) [inline core/src/mindustry/type/Item.java:20]:
```suggestion
    /** how electrically potent this item is. */
```

**Comment by Erisfiregamer1** [comment]
haha yes surge crate go zap zap

**Comment by Anuken** [comment]
This should be called "charge", not "discharge". "discharge" is more of a verb. 

Charge should also be part of unit explosion calculations.

**Comment by sk7725** [comment]
> Charge should also be part of unit explosion calculations.

Good idea. Surge suicide bombing would be a tactic but a fairly expensive one at least.

**Comment by sk7725** [comment]
I think I'm done.

**Pull request merged**